package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "bs-webapi",
-  "version": "0.18.0",
+  "version": "0.18.1",
```

Comment:
This is a breaking change, so it should be 0.19

Reply:
Agreed, @illusionalsagacity would you mind bumping to 0.19?